build fail in PatternMinerUTest

looks similar to main build fail?
```
[ 98%] Linking CXX executable PatternMinerUTest
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::match_results<__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<boost::sub_match<__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > >::maybe_assign(boost::match_results<__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<boost::sub_match<__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > > const&)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::raise_runtime_error(std::runtime_error const&)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::put_mem_block(void*)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::perl_matcher<__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<boost::sub_match<__gnu_cxx::__normal_iterator<char const*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > >, boost::regex_traits<char, boost::cpp_regex_traits<char> > >::construct_init(boost::basic_regex<char, boost::regex_traits<char, boost::cpp_regex_traits<char> > > const&, boost::regex_constants::_match_flags)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::get_mem_block()'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::verify_options(unsigned int, boost::regex_constants::_match_flags)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::get_default_error_string(boost::regex_constants::error_type)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::basic_regex<char, boost::regex_traits<char, boost::cpp_regex_traits<char> > >::do_assign(char const*, char const*, unsigned int)'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::cpp_regex_traits_implementation<char>::transform(char const*, char const*) const'
../../../opencog/learning/PatternMiner/libPatternMiner.so: undefined reference to `boost::re_detail::cpp_regex_traits_implementation<char>::transform_primary(char const*, char const*) const'
collect2: error: ld returned 1 exit status
tests/learning/PatternMiner/CMakeFiles/PatternMinerUTest.dir/build.make:119: recipe for target 'tests/learning/PatternMiner/PatternMinerUTest' failed
make[3]: *** [tests/learning/PatternMiner/PatternMinerUTest] Error 1
```

--- a/opencog/learning/PatternMiner/CMakeLists.txt
+++ b/opencog/learning/PatternMiner/CMakeLists.txt
@@ -17,6 +17,7 @@ IF (HAVE_cpprest)
         patternminer-types
         ${cpprest_LIBRARY}
         ${Boost_SYSTEM_LIBRARY}
+        ${Boost_REGEX_LIBRARY}
     )
 ELSE (HAVE_cpprest)
     ADD_LIBRARY (PatternMiner SHARED
@@ -31,6 +32,7 @@ ELSE (HAVE_cpprest)
         ${ATOMSPACE_LIBRARIES}
         patternminer-types
         ${Boost_SYSTEM_LIBRARY}
+        ${Boost_REGEX_LIBRARY}
     )
 ENDIF (HAVE_cpprest)
 
@@ -98,7 +100,3 @@ IF (HAVE_GUILE)
 ENDIF (HAVE_GUILE)
 
 # ------------------------------------------------------------
-
-
-
-

